### PR TITLE
Add Png crush optimizer

### PR DESCRIPTION
**Pull request by bjoernhaeuser**
From here: https://github.com/thumbor/thumbor/pull/428

### CODE REVIEW

File: thumbor_plugins/optimizers/pngcrush.py [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+    def should_run(self, image_extension, buffer):
+        return 'png' in image_extension and self.runnable
+
+    def optimize(self, buffer, input_file, output_file):
+        command = '%s -reduce -q %s %s ' % (
+            self.pngcrush_path,
+            input_file,
+            output_file,
+        )
+        os.system(command)
```
Comment (masom) [inline thumbor_plugins/optimizers/pngcrush.py]: `os.system` locks the GIL. Please use `subprocess.call`.

See https://github.com/thumbor/thumbor/blob/master/thumbor/optimizers/gifv.py#L33 and https://github.com/thumbor/thumbor/pull/436

Comment (bjoernhaeuser) [inline thumbor_plugins/optimizers/pngcrush.py]: Done :)

**Comment by bjoernhaeuser** [comment]
Could someone add travis?

**Comment by guilhermef** [comment]
Thanks @bjoernhaeuser 
Do you mind creating the test structure ?
I'll add it to travis now.

**Comment by bjoernhaeuser** [comment]
Will do!

**Pull request merged**